sdk/monitor/opentelemetry-exporter-azure-monitor/src/platform/nodejs/httpSender.ts

```diff
@@ -15,12 +15,15 @@ export class HttpSender implements Sender {
   private readonly _logger: Logger;
 
   private readonly _appInsightsClient: ApplicationInsightsClient;
+  private _appInsightsClientOptions: ApplicationInsightsClientOptionalParams;
 
-  constructor(
-    private _exporterOptions: Partial<AzureExporterInternalConfig> = {},
-    private _appInsightsClientOptions: ApplicationInsightsClientOptionalParams = {}
-  ) {
+  constructor(private _exporterOptions: Partial<AzureExporterInternalConfig>) {
     this._logger = this._exporterOptions.logger || new ConsoleLogger(LogLevel.ERROR);
+    // Build endpoint using provided configuration or default values
+    this._appInsightsClientOptions = {
+      endpoint: this._exporterOptions.endpointUrl + "/v" + this._exporterOptions.serviceApi
```

Comment:
Just checking - by the time we've gotten here is it guaranteed that endpointUrl will have been normalized (so, for instance, it doesn't already have a trailing slash)?

Reply:
yeah this feels a bit brittle

Reply:
Added code to format the URL in connectionString parsers, we were ignoring this before so we didn't realize it was not doing the formatting, connectionStrings from Azure portal actually include the slash at the end and plenty of spaces